packages/terra-application-layout/tests/jest/utilityHelpers.test.jsx

```diff
@@ -0,0 +1,108 @@
+import utilityHelpers from '../../lib/utils/utilityHelpers';
+
+describe('utilityHelpers', () => {
+  describe('getDefaultUtilityItems', () => {
+    it('returns the default set of utility items', () => {
```

Comment:
If this test ever fails, would it be easier to debug if we broke these tests out a bit more?

Reply:
Maybe, but the tests would still be very brittle.